sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -248,24 +248,25 @@ The following sections provide several code snippets covering some of the most c
 Text analytics support both synchronous and asynchronous client creation by using
 `TextAnalyticsClientBuilder`,
 
-<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L29-L32 -->
+<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L28-L31 -->
 ``` java
-TextAnalyticsClient textAnalyticsClient = new TextAnalyticsClientBuilder()
-    .subscriptionKey(SUBSCRIPTION_KEY)
-    .endpoint(ENDPOINT)
-    .buildClient();
+public void useSubscriptionKeySyncClient() {
```

Comment:
wrong line number

Reply:
there are more. But I stop adding comments here

Reply:
should be updated now :)